README.md

```diff
@@ -129,8 +129,26 @@ the steps accordingly.
 
 That's it!  The service should now be running and working.
 
-### Integration
+## Integration
 
 To integrate this service into an existing website, the easiest way is to implement it as an `iframe`:
 
     <iframe src="/node-monitor/" style="width:600px; height:500px; border:0"></iframe>
+
+## Development Virtual Environment
+
+There is prepared a Vagrant VM test-setup. Just follow these steps:
```

Comment:
This is an awkward formulation. What about "To set up a test VM using Vagrant, just follow these steps:"